### ISSUE TITLE
inconsistent suppress preprocessor for nowait?

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#606]
While sprinkling some suppression preprocessors around I noticed that there is a difference in the format of the preprocessors:

```
{&_proparse_prolint-nowarn(nowait)}
{&_proparse_ prolint-nowarn(noundo)}
{&_proparse_ prolint-nowarn(findnoerror)}
{&_proparse_ prolint-nowarn(varusage)}
```
The nowait preprocessor does not have a space before prolint.

**Comment by gquerret** [comment]
Typo in the documentation, that's now fixed

**Comment by stefandrissen** [comment]
It's not just a typo in the documentation (or the documentation is leading for rule suppression).

If I use {&_proparse _prolint-nowarn(nowait)} - then the issue is still flagged,
if I use {&_proparse_prolint-nowarn(nowait)} then the issue is suppressed.

**Comment by gquerret** [comment]
There's a subtle difference:
* `{&_proparse _prolint-nowarn(nowait)}` won't work
* `{&_proparse_ prolint-nowarn(nowait)}` will work

**Comment by gquerret** [comment]
`_proparse_` is the string which triggers a directive node instead of just preprocessor variable expansion

**Comment by stefandrissen** [comment]
Aha - thanks - that clears it up.
